pkgs/top-level/all-packages.nix

```diff
@@ -367,12 +367,17 @@ let
 
   fetchzip = import ../build-support/fetchzip { inherit lib fetchurl unzip; };
 
-  fetchFromGitHub = { owner, repo, rev, sha256, name ? "${repo}-${rev}-src" }: fetchzip {
+  fetchFromGitHub = { owner, repo, rev, sha256, name ? "${repo}-${rev}-src", host ? "github.com" }: fetchzip {
```

Comment:
Wouldn't this be more at home in a separate commit? It's more likely to go unnoticed this way.